blaze/compute/tests/test_pandas_compute.py

```diff
@@ -680,6 +680,21 @@ def test_str_lower():
     assert_series_equal(expected, result)
 
 
+class TestStrCat():
+    ds = dshape('3 * {name: string[10], comment: string[25], num: int32}')
+    s = symbol('s', dshape=ds)
+    data = [('alice', 'this is good', 0), ('suri', 'this is not good', 1),
+            ('jinka', 'this is ok', 2)]
+    df = pd.DataFrame(data, columns=['name', 'comment', 'num'])
+
+    def test_str_cat(self):
+        res = compute(self.s.name.str_cat(self.s.comment), self.df)
+        assert all(self.df.name.str.cat(self.df.comment) == res)
+
+    def test_str_cat_sep(self):
+        res = compute(self.s.name.str_cat(self.s.comment, sep=' -- '), self.df)
+        assert all(self.df.name.str.cat(self.df.comment, sep=' -- ') == res)
+
```

Comment:
Reused `dfbig` existing dataframe - also added a fixture for testing null values in rows. See latest changes.